.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+name: build
+# This workflow is  triggered on pushes, pull requests to the repository.
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+env:
+  version_in_development: v0.0.0
+permissions:
+  contents: write
+
+
+jobs:
+  draft-release:
+    name: Draft Release if develop branch
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.draft_release.outputs.upload_url }}  
+      release_id: ${{ steps.draft_release.outputs.id }}  
+    steps:
+      - name: Create Release
+        id: draft_release
+        if: github.ref == 'refs/heads/main' || github.head_ref == 'feature/add-script'
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          owner: tipi-build 
+          repo: re-zipper-package
+          commitish: main
+          tag_name: ${{ env.version_in_development }}
+          release_name: ${{ env.version_in_development }} ${{ github.sha }}
+          draft: true
+          prerelease: true
+
+  build-linux:
+    name: build-linux
+    runs-on: ubuntu-latest
+    needs: draft-release
+    steps:
+      - uses: actions/checkout@v3
+      - name: install and build 
+        run: |
+          sudo apt update && sudo apt install -y zip wget 
```

Comment:
Is curl installed? If so, could you use that?

Reply:
zip and wget are already install too. It's just to make sure that whatever machine it's running on, I've got it. Why do you want me to use curl?

Reply:
I was under the impression they weren't present. It's odd to be installing deps that are already present